kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java

```diff
@@ -144,9 +144,9 @@ private void handleKeyReleased(KeyEvent keyEvent) {
     if (keyEvent.getCode() == KeyCode.ESCAPE) {
       // clear the current sketch and start a new sketch
       startSketch();
-    } else if (keyEvent.getCode() == KeyCode.ENTER && sketchEditor.isSketchValid()) {
+    } else if (keyEvent.getCode() == KeyCode.ENTER) {
       // project the sketched geometry to WGS84 to comply with the KML standard
-      Geometry sketchGeometry = sketchEditor.getGeometry();
+      Geometry sketchGeometry = geometryEditor.getGeometry();
       Geometry projectedGeometry = GeometryEngine.project(sketchGeometry, SpatialReferences.getWgs84());
```

Comment:
Maybe one to think about/discuss - I'm not sure what's better but just jumped out. There is the use of the term "sketch" here. Is that still a term we want to use? Reading the doc my initial thought is to use `createGeometry()` in place of `startSketch()` and e.g. `geometry` in place of `sketchGeometry`.
There's also a few uses of "sketch" in the first couple of sections of the Readme. Maybe we don't mind but one to consider!

Reply:
Very good point, I was thinking about these as well. I agree with you these soun weird in the new context, changed them now.